bootstrap/kubeadm/types/v1beta2/bootstraptokenstring_test.go

```diff
@@ -185,26 +173,15 @@ func TestNewBootstrapTokenString(t *testing.T) {
 	for _, rt := range tests {
 		t.Run(rt.token, func(t *testing.T) {
 			actual, err := NewBootstrapTokenString(rt.token)
-			if (err != nil) != rt.expectedError {
-				t.Errorf(
-					"failed NewBootstrapTokenString for the token %q\n\texpected error: %t\n\t  actual error: %v",
-					rt.token,
-					rt.expectedError,
-					err,
-				)
-			} else if !reflect.DeepEqual(actual, rt.bts) {
-				t.Errorf(
-					"failed NewBootstrapTokenString for the token %q\n\texpected: %v\n\t  actual: %v",
-					rt.token,
-					rt.bts,
-					actual,
-				)
-			}
+			g.Expect(err != nil).To(Equal(rt.expectedError))
```

Comment:
I think we can be a bit clearer with a bit of extra verbosity here:
```suggestion
			if rt.expectedError {
				g.Expect(err).To(HaveOccurred())
			} else {
				g.Expect(err).NotTo(HaveOccurred())
			}
```

Reply:
done